shared/fixes/bash/network_ipv6_privacy_extensions.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_rhel
+
+# enable randomness in ipv6 address generation
+for interface in $(ls /etc/sysconfig/network-scripts/ifcfg-*)
```

Comment:
Just writing `/etc/sysconfig/network-scripts/ifcfg-*` is better than the `ls` stuff as it is simple and handles names with spaces correctly: http://mywiki.wooledge.org/ParsingLs